### PR TITLE
Test auth middleware#141

### PR DESCRIPTION
**Pull request by MichaelCaraccio**

### CODE REVIEW

**Comment by MichaelCaraccio** [comment]
The Admin middleware test will come after, I have already created a branch for the issues "Admin Dashboard". This issue will have its middleware test too.

**Comment by ice-blaze** [comment]
If the class admin is never called, you can comment it. Then for your next issue where you will need it, uncomment it. #team100%

File: app/Http/Middleware/RedirectIfAuthenticated.php [hunk 1]
```diff
@@ -18,9 +18,9 @@ class RedirectIfAuthenticated
      */
     public function handle($request, Closure $next, $guard = null)
     {
-        if (Auth::guard($guard)->check()) {
+        /*if (Auth::guard($guard)->check()) {
```
Comment (ice-blaze) [inline app/Http/Middleware/RedirectIfAuthenticated.php]: maybe put somehting in comment like `TODO original code` or just a simple comment without the todo

File: tests/MiddlewareTest.php [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            ['GET', '/chat/'.$chat->id.'/edit', 200, 'auth'],
+            ['GET', '/projects/create', 200, 'auth'],
+            ['GET', '/projects/'.$project->id.'/edit', 200, 'auth'],
+            ['GET', '/comments/'.$comment_private->id.'/edit', 200, 'auth'],
+            ['GET', '/projects/'.$project->id.'/privateComments', 200, 'auth'],
+            ['GET', '/projects/'.$project->id.'/invitations', 200, 'auth'],
+            ['GET', '/projects/'.$project->id.'/join', 200, 'auth'],
+            ['GET', '/talents/'.$user->id.'/invitations', 200, 'auth'],
+            ['GET', '/talents/'.$recruit->id.'/recruit', 200, 'auth'],
+            //['GET', '/projects', 200, 'auth'],
```
Comment (ice-blaze) [inline tests/MiddlewareTest.php]: Why is it commented ?

**Comment by MichaelCaraccio** [comment]
WOW achievement unlocked

**Pull request merged**